spec/integration_spec.rb

```diff
@@ -1,25 +1,60 @@
 require 'spec_helper'
+require 'liquid'
 
-Template = <<-TEMPLATE
-  <h1>{{ title | capitalize }}<h1>
-  <p>
-    This is a pretty slick Liquid page.
-    It isn't vulnerable to <%= erb %> injection
-  </p>
-  {% tag 'tags', 'still', 'work' %}
-  {% tag 'tags', 'still', 'work' %}
-  {% tag 'tags', 'still', 'work' %}
-  {% block nested.variable %}
-    variables can be resolved
-    Uh oh!
-  {% endblock %}
-TEMPLATE
+module CapitalizeFilter
+  def capitalize(input)
+    input.upcase
+  end
+end
+
+class LiquidContext
+  include CapitalizeFilter
+
+  def initialize(context)
+    @context = context
+  end
+
+  def get_binding
+    binding
```

Comment:
I though we wanted to avoid going through `binding` and just use `Hash#[]` and `Hash#[]=`